app/spiffworkflow/messages/propertiesPanel/MessageSelect.js

```diff
@@ -35,6 +35,9 @@ export function MessageSelect(props) {
           commandStack.execute('element.updateModdleProperties', {
             element: shapeElement,
             moddleElement: businessObject,
+            properties: {
```

Comment:
No idea why this was not here - but it how we would connect a message start event to the message it will be started by - I wonder if these lines were not deleted at some point.